src/docs/quick-start.md

```diff
@@ -90,11 +92,66 @@ width={836} height={214} quality={100} />
 
 **You can now run `railway open` and you will taken to your [Project Canvas](/quick-start#the-canvas)**.
 
+## Deploying Your Project - From a Docker Image
+
+Railway supports deploying pre-built Docker images from the following registries:
+
+- <a href="https://hub.docker.com" target="_blank">Docker Hub</a>
+- <a href="https://ghcr.io" target="_blank">GitHub Container Registry</a>
+- <a href="https://quay.io" target="_blank">RedHat Container Registry</a>
+- <a href="https://docs.gitlab.com/ee/user/packages/container_registry" target="_blank">GitLab Container Registry</a>
+
+To get started deploying a Docker image, we will first make a new <a href="/overview/the-basics#project--project-canvas" target="_blank">project</a>.
+
+- Open up the <a href="/overview/the-basics#dashboard--projects" target="_blank">dashboard</a> → Click **New Project**.
+
+- Choose the **Empty project** option.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/f_auto,q_auto/v1727281981/docs/quick-start/emptyproject_q8vqfz.png"
+alt="screenshot of new project menu with deploy from github selected"
+width={836} height={714} quality={100} />
+
+After the project is created, you will land on the <a href="/quick-start#the-canvas" target="_blank">Project Canvas</a>. A panel will appear prompting you to Add a Service.
+
+<Image src="https://res.cloudinary.com/railway/image/upload/f_auto,q_auto/v1727281215/docs/quick-start/add_a_service.png"
+alt="screenshot of add a service panel on the canvas"
+layout="responsive"
+width={1422} height={1284} quality={100} />
+
+- Click **Add a Service** and select **Docker Image** option from the modal that pops up.
```

Comment:
Perhaps - "and select the **Docker Image** option"

I think the addition of "the" makes the sentence flow better.

Reply:
I also agree. Had it there but when I ran it through a grammar correction service, it suggested removing "the". So weird 😆

Reply:
Adding it. It makes it flow better like you mentioned.